src/main/java/net/imglib2/type/numeric/integer/UnsignedVariableBitLengthType.java

```diff
@@ -280,6 +280,16 @@ public void mul( final UnsignedVariableBitLengthType t )
 		set( get() * t.get() );
 	}
 
+	@Override
+	public void pow( final UnsignedVariableBitLengthType c) {
+		set( ( long )Math.pow( get(), c.get() ) );
```

Comment:
Same question as above, shouldn't we leave the casting to the `setReal(double)` method? I would expect rounding.

Reply:
Yes, agreed.